screens/Home.js

```diff
@@ -1,60 +1,79 @@
-import React, { Component } from 'react';
-import { Text, View, StyleSheet } from 'react-native';
+import React, { Component } from "react";
+import { Text, View, StyleSheet } from "react-native";
+import getAllData from "../getCMS";
```

Comment:
unused import - this is why I think we should be saving to json btw - ask rahul about imports for this kind of data